lms/djangoapps/instructor/views/api.py

```diff
@@ -1371,6 +1371,7 @@ def get_proctored_exam_results(request, course_id):
     return JsonResponse({"status": success_status})
 
 
+@transaction.non_atomic_requests
```

Comment:
Why this was added?

Reply:
https://github.com/edx/edx-platform/blob/240a8c2e1a1ad6f56fc5c12f0c424531c459bbaa/common/djangoapps/util/db.py#L97-L126